ci/init.js

```diff
@@ -44,6 +46,10 @@ if (isJestWorker) {
 if (shouldInit) {
   tracer.init(options)
   tracer.use('fs', false)
+  if (isTrue(process.env.DD_CIVISIBILITY_MANUAL_API_ENABLED)) {
```

Comment:
Is this standardized across languages?

Reply:
it isn't: nodejs' is the only tracer doing this. I still think it's a good idea to do a first release behind a feature flag

Reply:
Oh yeah I meant the name of the environment variable. If it only applies to Node then it's fine 👍